doc/classes/Variant.xml

```diff
@@ -5,20 +5,30 @@
 	</brief_description>
 	<description>
 		In computer programming, a Variant class is a class that is designed to store a variety of other types. Dynamic programming languages like PHP, Lua, JavaScript and GDScript like to use them to store variables' data on the backend. With these Variants, properties are able to change value types freely.
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var foo = 2 # foo is dynamically an integer
 		foo = "Now foo is a string!"
 		foo = Reference.new() # foo is an Object
 		var bar: int = 2 # bar is a statically typed integer.
 		# bar = "Uh oh! I can't make static variables become a different type!"
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		// ... but C# is statically typed. Once a variable has a type it cannot be changed. All types and classes inherit object. However you can use the var keyword in methods to let the compiler decide the type automatically.
```

Comment:
```suggestion
		// ... but C# is statically typed. Once a variable has a type it cannot be changed. However you can use the var keyword in methods to let the compiler decide the type automatically.
```

Struct types do not inherit object.

Reply:
Oh true. I mixed the types with their helper classes